python/ray/tune/commands.py

```diff
@@ -162,16 +164,30 @@ def list_trials(experiment_path,
 
     print_format_output(checkpoints_df)
 
+    if output:
+        experiment_path = os.path.expanduser(experiment_path)
+        output_path = os.path.join(experiment_path, output)
+        file_extension = os.path.splitext(output)[1].lower()
+        if file_extension in (".p", ".pkl", ".pickle"):
+            checkpoints_df.to_pickle(output_path)
+        elif file_extension == ".csv":
+            checkpoints_df.to_csv(output_path, index=False)
```

Comment:
can this be a helper function that you reuse in both ls, lsx calls?

Reply:
Actually, maybe we don't need to do this in this PR and might just be overabstracting